header.php

```diff
@@ -52,11 +52,13 @@
 
   $logo_id = IGV_get_option('_igv_social_options', '_igv_metadata_logo_id');
```

Comment:
I wonder if this needs a new metabox as it isn't nec the socal media logo anymore [that logo file being used for microdata mainly and probably wanting to me the plain black on white logo. But maybe the logo shown in the skin is now the new logo and it still works. But then again would be nicer to use a SVG for the new logo rather than raster file.